### DISCUSSION TITLE
Please release Phoenix Synapse~~~~~~

### DISCUSSION THREAD

**Discussion by yjoonh** [phcode-dev/phoenix#1758] [General]
Phoenix Synapse... Please release this extension.

Brackets Synapse: https://github.com/kooohei/brackets-synapse

**Comment by abose** [comment]
We will be adding native support for ftp/sftp in the coming releases.

**Reply by abose** [reply]
It may not come this year. Git and terminal integration is taking most of our time.

**Reply by devvaannsh** [reply]
@yjoonh, not yet! We'll work on it, but right now our complete focus is on terminal and AI integration as they are the most requested features as of now. But, we'll start working on it soon. Apologies!
